zipline/__main__.py

```diff
@@ -210,6 +213,11 @@ def _(*args, **kwargs):
     metavar='DIRNAME',
     help='Directory where the realtime collected minutely bars are saved'
 )
+@click.option(
+    '--broker-list',
```

Comment:
Could you please rename this option (and the variables downstream) to '--list-brokers' ? I think it shows better the intent.